### PR TITLE
capi-add-dashboard-to-alerts

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/roadmap/issues/3174

add grafana dashboards to CAPI/CAPA alerts

### CODE REVIEW

File: helm/prometheus-rules/templates/alerting-rules/capa.management-cluster.rules.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ spec:
       annotations:
         description: '{{`Container {{ $labels.container }} in pod {{ $labels.namespace }}/{{ $labels.pod }} is restarting too often.`}}'
         opsrecipe: container-is-restarting-too-often/
+        dashboard: bdiako8tt1b7kc/capa-agregated-error-logs-for-capa-controllers
```
Comment (whites11) [inline helm/prometheus-rules/templates/alerting-rules/capa.management-cluster.rules.yml:35]: what is the horrible random string?

**Review by whites11 — CHANGES_REQUESTED** [review]
please add changelog entry, thanks

File: helm/prometheus-rules/templates/alerting-rules/capa.management-cluster.rules.yml (hunk 1 above)
Comment (calvix) [reply helm/prometheus-rules/templates/alerting-rules/capa.management-cluster.rules.yml:35]: randomly generated dashboard ID

Comment (QuentinBisson) [reply helm/prometheus-rules/templates/alerting-rules/capa.management-cluster.rules.yml:35]: You can configure a proper uid instead :)

**Pull request merged**